packages/flutter_tools/test/commands.shard/hermetic/pub_get_test.dart

```diff
@@ -77,6 +77,33 @@ void main() {
     FileSystem: () => fileSystem,
   });
 
+  testUsingContext('pub get on target directory', () async {
+    fileSystem.currentDirectory.childDirectory('target').createSync();
+    final Directory targetDirectory = fileSystem.currentDirectory.childDirectory('target');
+    targetDirectory.childFile('pubspec.yaml').createSync();
+    targetDirectory.childFile('.flutter-plugins').createSync();
+    targetDirectory.childFile('.flutter-plugins-dependencies').createSync();
+    targetDirectory.childFile('.packages').writeAsBytesSync(<int>[0]);
+    targetDirectory.childFile('.dart_tool/package_config.json')
+      ..createSync(recursive: true)
+      ..writeAsBytesSync(<int>[0]);
+
+    final PackagesGetCommand command = PackagesGetCommand('get', false);
+    final CommandRunner<void> commandRunner = createTestCommandRunner(command);
+
+    await commandRunner.run(<String>['get', targetDirectory.path]);
+
+    expect(await command.usageValues, const CustomDimensions(
```

Comment:
Should we check that `.dart_tool/package_config.json` got written?

Reply:
I think you can delete this expectation actually, as its redundant with the previous test